examples/apollo-engine/README.md

```diff
@@ -96,14 +93,13 @@ const resolvers = {
 
 const server = new GraphQLServer({ typeDefs, resolvers })
 
-const engine = new Engine({
-  engineConfig: { apiKey: process.env.APOLLO_ENGINE_KEY },
-  endpoint: '/',
-  graphqlPort: parseInt(process.env.Port, 10) || 4000,
-})
-engine.start();
+// Enable gzip compression
+// ref: https://www.apollographql.com/docs/engine/setup-node.html#enabling-compression
+server.express.use(compression())
```

Comment:
are you sure this still works? That information disappeared and personally had issues with Engine stripping gzip: https://github.com/apollographql/apollo-engine-js/issues/155

Reply:
Interesting. I had tried it and it worked for me.